Vagrantfile

```diff
@@ -98,6 +98,16 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     backend.vm.provision :shell, :inline => "puppet apply /vagrant/tests/sensu-backend-federated-cluster.pp"
   end
 
+  config.vm.define "el8-agent", autostart: true do |agent|
+    agent.vm.box = "centos/8"
+    agent.vm.box_url = "http://cloud.centos.org/centos/8/x86_64/images/CentOS-8-Vagrant-8.1.1911-20200113.3.x86_64.vagrant-virtualbox.box"
```

Comment:
This should have a `# TODO` comment with the link to the issue and mention that after centos 8.1, this can likely be removed.

Reply:
Comments added

Reply:
I don't see the comment, perhaps you forgot to push your changes up.

Reply:
I pushed just now, forgot to amend the commit.